app/main.py

```diff
@@ -1,13 +1,30 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+
+    return {
+        "line": error.get("line_number"),
+        "column": error.get("column_number"),
+        "message": error.get("text"),
+        "name": error.get("code"),
+        "source": "flake8"
+    }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+
+    return {
+        "errors":
+            [
+                format_linter_error(dc)
+                for dc in errors
+            ],
+        "path": file_path,
+        "status": "failed" if any(errors) else "passed"
```

Comment:
remove redundant "any"
```suggestion
        "status": "failed" if errors else "passed"
```